manifests/install/charts/as-a-second-scheduler/README.md

```diff
@@ -1 +1,45 @@
+| --------------------------------------- | ----------------------------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------- |
+| `scheduler.name`                        | Scheduler name                                                                                                                            | `scheduler-plugins-scheduler`                           |
+| `scheduler.image`                       | Scheduler image                                                                                                                           | `k8s.gcr.io/scheduler-plugins/kube-scheduler:v0.22.6`   |
+| `scheduler.namespace`                   | Scheduler namespace                                                                                                                       | `scheduler-plugins`                                     |
+| `scheduler.replicaCount`                | Scheduler replicaCount                                                                                                                    | `1`                                                     |
+| `controller.name`                       | Controller name                                                                                                                           | `scheduler-plugins-controller`                          |
+| `controller.image`                      | Controller image                                                                                                                          | `k8s.gcr.io/scheduler-plugins/controller:v0.22.6`       |
+| `controller.namespace`                  | Controller namespace                                                                                                                      | `scheduler-plugins`                                     |    
+| `controller.replicaCount`               | Controller replicaCount                                                                                                                   | `1`                                                     |
+| `plugins.enabled`                       | Plugins enabled                                                                                                                           | `["Coscheduling","CapacityScheduling","NodeResourceTopologyMatch", "NodeResourcesAllocatable"]` |
+| `global.queueSort`                      | global queueSort, needs to be globally enabled once                                                                                       | `["Coscheduling"]`                                      |
+| `global.extensions.preFilter`           | global extensions preFilter                                                                                                               | `["Coscheduling", "CapacityScheduling"]`                |
```

Comment:
The description should be something like:
 ```markdown
 `preFilter` extension config. This is be used if the `preFilter` plugin is enabled.
 ```

Please apply the same for the rest of all extensions. parameters